docs/contributing.md

```diff
@@ -25,9 +25,10 @@ Below are a few different queries you can use to find appropriate issues to work
 
 
 ## Sending Pull Requests ##
-
 Patches should be submitted as [pull requests](https://help.github.com/articles/about-pull-requests/) (PRs).
 
+> When submitting PRs, We expect external contributors to push patches to a [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks) of [`application-services`](https://github.com/mozilla/application-services). For more information about submitting PRs from forks, read [GitHub's guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
```

Comment:
slightly surprised this is a blockquote, but assuming that's intentional, 👍

Reply:
Yeah, on the website they end up looking like notes 🤷